src/utils/feature-flags.ts

```diff
@@ -101,10 +101,21 @@ export const FF_DEV_2461 = "fflag_feat_front_dev_2461_audio_paragraphs_seek_chun
  */
 export const FF_DEV_2918 = "fflag_fix_front_dev_2918_labeling_filtered_paragraphs_250822_short";
 
+/**
+ * Disallow drawing regions outside of the video canvas
+ * Also disables offscreen zooming and panning
+ * @link https://app.launchdarkly.com/default/production/features/fflag_fix_front_dev_3350_restrict_drawing_area_short
```

Comment:
Not sure it's a good idea to hardcode such links, especially in opensource. This page can be easily found by ff name and the whole comment is too big. I'd suggest to use simple one-line comment.